docs/pages/blog/2020-developer-survey-results.md

```diff
@@ -0,0 +1,303 @@
+---
+description: 2020 Material-UI Developer Survey results
+---
+
+# 2020 Material-UI Developer Survey results
+
+While we are currently working on the upcoming release of Material-UI v5, we need to prioritize our
+roadmap for the coming year. To refine our focus, we launched a developer survey few months back,
+to which we received 1488 contributions. This year we had twice as more contributions than last year,
+so thank you for your participation!
+The survey is now closed and this is a summary of the results.
+
+The survey was broken into three sections, "Introduction", "About you" and "Your product".
+
+## Introduction
+
+Here we asked for the good, the bad and the ugly. Thankfully, it's mostly good, but we can focus
+on the things that will make it even better.
```

Comment:
Could we have some new commentary throughout, rather than copying last years? I don't know how many developers will notice (or care), but it'll get a negative score as duplicate content from an SEO perspective.

Reply:
Updated the text, please take a look again.